pkg/schedule/metrics.go

```diff
@@ -40,10 +40,19 @@ var (
 			Name:      "patrol_regions_time",
 			Help:      "Time spent of patrol checks region.",
 		})
+
+	patrolCheckRegionsChanLenGauge = prometheus.NewGauge(
```

Comment:
Do we need to update `pd.json` metric?

Reply:
No, I'm even thinking about removing it.